Firestore/core/test/unit/local/target_cache_test.cc

```diff
@@ -18,6 +18,7 @@
 
 #include <set>
 #include <string>
+#include <unordered_set>
```

Comment:
is this needed?

Reply:
There are use cases of `unordered_set` inside `target_cache_test.cc`. Not sure why the compiler doesn't complain before, but it complains in this pr without include statement.